backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewEdgeToolSwitch.java

```diff
@@ -221,6 +225,10 @@ private EClassifier getDefinitionFromUsage(Usage object) {
         return SysmlPackage.eINSTANCE.getEClassifier(definitionFromUsage);
     }
 
+    private boolean isA(NodeDescription nd, EClass eClass) {
```

Comment:
If you just want to know if the domain type of the NodeDescription is of the given eClass, you could also do this:
`SysMLMetamodelHelper.buildQualifiedName(eClass).equals(nd.getDomainType())`

Reply:
My method does not have the correct name. It would rather be something like `isTheNodeDescriptionFor`

Reply:
I have used this tooling function elsewhere to increase the readability of the code (almost all caseXXX filter on specific node descriptions).